src/actions/guides/database/database_setup.cr

```diff
@@ -127,6 +128,7 @@ class Guides::Database::DatabaseSetup < GuideAction
     > If you need to re-seed, you can run `lucky db.drop` and then `./script/setup` to re-create
     > and seed your local database.
 
+    #{permalink(ANCHOR_MULTIPLE_DATABASES)}
```

Comment:
Out of curiosity, why all the special perma links? does this markdown parser not create ids for the headers?

Reply:
It does, I forget the issue I ran in to, but it was a hack around the issue. Then we decided to use constants so we get the compile errors if they go missing.